systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1460,6 +1463,69 @@ void checkStorageSizeForVolumes(List<PersistentVolumeClaim> volumes, String[] di
         }
     }
 
+    @Tag(SCALABILITY)
+    @Test
+    void testBigAmountOfTopics() {
+        int numberOfTopics = 300;
+
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 3, 1).done();
+
+        for (int i = 0; i < numberOfTopics; i++) {
+            String topicName = "topic-example" + i;
+            testMethodResources().topic(CLUSTER_NAME, topicName, 3, 3).done();
+            LOGGER.info("Checking that topic {} was created", topicName);
+            assertThat(cmdKubeClient().list("kafkatopic"), hasItems(topicName));
```

Comment:
This sounds as if it is just testing that the `KafkaTopic` CR exists. I think this needs to actually connect to Kafka and check that the topic was created there and has the expected settings.